src/plugins/dashboard/public/application/top_nav/options.tsx

```diff
@@ -101,12 +111,24 @@ export class OptionsMenu extends Component<Props, State> {
           />
         </EuiFormRow>
 
+        <EuiFormRow>
+          <EuiSwitch
+            label={i18n.translate('dashboard.topNav.options.syncCursorBetweenPanelsSwitchLabel', {
+              defaultMessage: 'Sync cursor across panels',
+            })}
+            checked={this.state.syncCursor}
+            onChange={this.handleSyncCursorChange}
+            data-test-subj="dashboardSyncCursorCheckbox"
+          />
+        </EuiFormRow>
+
```

Comment:
**nit:** It feels like the options menu is getting a bit messy - perhaps we could organize all the "Sync X" options under a different form row so that it can have its own title? 

image.png

Reply:
This is a great idea for now! Eventually we should start thinking of a design which will allow us to offload the options UI, and an ability to grab options in a generic way to other plugins. This could be accomplished with a `dashboardOptions` registry, and some mechanism by which the dashboard hands those options and their values down to its child embeddables.

Reply:
Thanx for the tip @Heenawter, done